src/app/screens/Send/index.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { act, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { MemoryRouter } from "react-router-dom";
+import { settingsFixture as mockSettings } from "~/../tests/fixtures/settings";
+import * as SettingsContext from "~/app/context/SettingsContext";
+
+import Send from "./index";
+
+jest.spyOn(SettingsContext, "useSettings").mockReturnValue({
+  settings: mockSettings,
+  isLoading: false,
+  updateSetting: jest.fn(),
+});
+
+describe("Send", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("renders via Send (popup)", async () => {
+    const user = userEvent.setup();
+    render(
+      <MemoryRouter>
+        <Send />
+      </MemoryRouter>
+    );
+
+    expect(
+      await screen.getByLabelText("Lightning Invoice")
+    ).toBeInTheDocument();
+    await act(async () => {
+      await user.type(
+        screen.getByLabelText("Lightning Invoice"),
+        "    sampleinvoice  "
```

Comment:
Can we add the `lightning:` here as well? Then it's done. Thanks for you work!

Reply:
Done!